### ISSUE TITLE
Incompatible with next/image

### ISSUE THREAD

**Issue by hutber** [cedricdelpoux/react-responsive-masonry#27]
Thanks for putting this together, I just wanted to flag that sadly it does not work with next/image. I assumed this was due to lazy loading, but I have turned this off and everything renders, but the styles have everything set to height 0.

I did force all parents to be 100vh, and this was the outcome:

![image](https://user-images.githubusercontent.com/1344898/104121822-373e4080-5339-11eb-8af7-473ddf2a7d30.png)

No idea if you want to action this in anyway, but thought I would flag it.

**Comment by panoslympe** [comment]
Any solution to this yet? :D

**Comment by mpetricek** [comment]
Follow this issue https://github.com/vercel/next.js/discussions/18739#discussioncomment-344932

**Comment by omarel1** [comment]
Yes I had the same problem and had to use the the img tag not NextJS image component.

**Comment by cedricdelpoux** [comment]
This is fixed on master: Masonry now rebalances the columns when a child's height changes after it was placed, like an image finishing its load. Thanks for reporting it!

**Comment by cedricdelpoux** [comment]
This shipped in v2.9.0, so you can upgrade whenever you're ready.
